lib/Data/Chronicle/Reader.pm

```diff
@@ -205,6 +205,35 @@ sub get_for_period {
     return \@result;
 }
 
+=head3 C<< my $data = get_history("category1", "name1", 1) >>
+
+Query Pg archive for the data under "category1::name1" at the provided number of revisions in the past.
+
+=cut
+
+sub get_history {
+    my $self     = shift;
```

Comment:
why not join to one line ? 
```my ($self, $category, $name, $rev) = @_;```

Reply:
Just following the style of the rest of the package.